.github/workflows/daily.yml

```diff
@@ -261,3 +261,27 @@ jobs:
       run: ./runtest-sentinel
     - name: cluster tests
       run: ./runtest-cluster
+
+  redis-test-ubuntu-jemalloc:
+    runs-on: ubuntu-latest
+    if: github.repository == 'redis/redis'
+    timeout-minutes: 14400
+    steps:
+    - uses: actions/checkout@v2
+    - name: make
+      run: make REDIS_CFLAGS='-Werror -DREDIS_TEST'
+    - name: test
+      run: ./src/redis-server test all
+
+  redis-test-ubuntu-32bit:
+    runs-on: ubuntu-latest
+    if: github.repository == 'redis/redis'
+    timeout-minutes: 14400
+    steps:
+    - uses: actions/checkout@v2
+    - name: make
+      run: |
+        sudo apt-get update && sudo apt-get install libc6-dev-i386
+        make 32bit REDIS_CFLAGS='-Werror -DREDIS_TEST'
+    - name: test
+      run: ./src/redis-server test all
```

Comment:
i think these can be merged into the existing jobs.
i.e. find 3 of the jobs that build and test redis (normal, 32bit and valgrind), add the CFLAGS, and an extra step at the end.

Reply:
I will make the change tomorrow.